darwin-amd64/bindings_arrow.go

```diff
@@ -167,16 +168,19 @@ func DataChunkFromArrow(conn Connection, rec arrow.RecordBatch, schema ArrowConv
 	}()
 	cdata.ExportArrowRecordBatch(rec, (*cdata.CArrowArray)(arr), (*cdata.CArrowSchema)(arrs))
 
-	var chunk C.duckdb_data_chunk
-	ed := C.duckdb_data_chunk_from_arrow(conn.data(), (*C.struct_ArrowArray)(arr), schema.data(), &chunk)
+	cd := chunk.data()
+	ed := C.duckdb_data_chunk_from_arrow(conn.data(), (*C.struct_ArrowArray)(arr), schema.data(), &cd)
 	errData := ErrorData{Ptr: unsafe.Pointer(ed)}
 	if debugMode && ed != nil {
 		incrAllocCount("errorData")
 	}
-	if debugMode && chunk != nil {
+	if debugMode {
```

Comment:
remove the increase counter for the chunk, now it is provided as arguments. This causes panic in tests with the debug_bindings build flag.